### PR TITLE
Improve German translations slightly

### PR DESCRIPTION
**Pull request by soebbing**
This change improves the German translations by fixing a few typos.

## Description
Just the aforementioned typo fixes. :)

## Motivation and Context

I love the project and always feel a slight sting when I read one of the messages with a typo. 
- [ ] I have raised an issue to propose this change ([required](https://github.com/evilsocket/pwnagotchi/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

It hasn't as of yet. Please let me know how to test the change. I know how to code, I'm just not familiar with Python and it's ecosystem.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/evilsocket/pwnagotchi/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

File: pwnagotchi/locale/de/LC_MESSAGES/voice.po [hunk 1]
```diff
@@ -174,10 +174,10 @@ msgstr "Cool, wir haben {num} neue Handshake{plural}!"
 
 #, python-brace-format
 msgid "You have {count} new message{plural}!"
-msgstr "Cool, wir haben {num} neue Handshake{plural}!"
```
Comment (benleb) [inline pwnagotchi/locale/de/LC_MESSAGES/voice.po]: I think "Handshakes" would be the better word, even for german pwnagotchi users.

**Comment by soebbing** [comment]
After reading a bit more in the docs I found the commands to update and compile the language files as well :)

File: pwnagotchi/locale/de/LC_MESSAGES/voice.po [hunk 2]
```diff
@@ -62,6 +69,13 @@ msgstr "Ich bin sehr traurig..."
 msgid "I'm sad"
 msgstr "Ich bin traurig"
 
+#, fuzzy
+msgid "Leave me alone ..."
+msgstr "Lass mich in Ruhe..."
```
Comment (mrseeker) [inline pwnagotchi/locale/de/LC_MESSAGES/voice.po]: Is "Lass mich allein ..." not a more appropiate translation for this?

File: pwnagotchi/locale/de/LC_MESSAGES/voice.po [hunk 3]
```diff
@@ -62,6 +69,13 @@ msgstr "Ich bin sehr traurig..."
...
+
+msgid "I'm mad at you!"
+msgstr "Ich bin wütend auf Dich!"
```
Comment (mrseeker) [inline pwnagotchi/locale/de/LC_MESSAGES/voice.po]: This does not really express anger...

File: pwnagotchi/locale/de/LC_MESSAGES/voice.po (hunk 2 above)
Comment (soebbing) [reply pwnagotchi/locale/de/LC_MESSAGES/voice.po]: "Verpiss Dich!" might be a bit more aggressive?

**Comment by evilsocket** [comment]
This is not an improvement, closing.

File: pwnagotchi/locale/de/LC_MESSAGES/voice.po (hunk 3 above)
Comment (soebbing) [reply pwnagotchi/locale/de/LC_MESSAGES/voice.po]: "Ich bin echt sauer auf Dich!" - a bit more colloquial

**Comment by soebbing** [comment]
Am I missing something? Fixing typos and adding missing translation keys is not an improvement? It isn't a big addition, but no improvement at all?

**Comment by evilsocket** [comment]
changing words from handshake to message is not fixing typos

**Comment by soebbing** [comment]
I thought this key was for new messages, not for new handshakes (the one above seems to be for new handshakes). If I am mistaken, I'll revert this change happily. What about all the other changes?

**Pull request closed without merging**